examples/tools/rectangle-scissors.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: segmentation-tool
```

Comment:
Maybe we stop maintaining individual examples? A polished example that we use for netlify testing is much more likely to be maintained, and should be good enough. Worth noodling on.

Reply:
More talk about examples.